.github/workflows/ett.yml

```diff
@@ -117,119 +117,81 @@ jobs:
           path: ./dist/apps/ett/${{ env.app-name }}/
           if-no-files-found: error
 
-  container-pr:
-    name: 'PR: Host container'
-    if: ${{ github.event_name  == 'pull_request' }}
-    needs: app
+  define_build:
+    name: Define build matrix
+    needs: affected
     runs-on: ubuntu-latest
-    env:
-      host-image-tag: ${{ format('pr-{0}', github.event.pull_request.number) }}
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/download-artifact@v2
-        with:
-          name: ${{ env.app-name }}
-          path: ./dist/apps/ett/${{ env.app-name }}/
-      - name: Container image name
-        id: image-name
-        uses: ASzc/change-string-case-action@v2
-        with:
-          # TODO(xlgni): coordinate name change to `env.app-name` with Fenris infrastructure
-          string: ghcr.io/${{ github.repository }}-ett-app
-      - name: Build host container image
-        run: docker build ./ --file ./infrastructure/eo/host/Dockerfile --tag ${{ steps.image-name.outputs.lowercase }}:${{ env.host-image-tag }}
-      - name: Publish host container image
-        run: |
-          docker login ghcr.io --username ${{ github.actor }} --password ${{ github.token }}
-          docker push ${{ steps.image-name.outputs.lowercase }}:${{ env.host-image-tag }}
-
-  container_merge:
-    name: 'Merge: Host container'
-    if: ${{ github.ref == 'refs/heads/main' }}
-    needs: app
     outputs:
-      release_tag: ${{ steps.build_and_push.outputs.RELEASE_VERSION }}
-    runs-on: ubuntu-latest
+      chart_version: ${{ steps.get_semver.outputs.semver }}
+      containers: >-
```

Comment:
Because of the way yaml works: https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines, removes the feed line from the input.

    >, |: "clip": keep the line feed, remove the trailing blank lines.
    >-, |-: "strip": remove the line feed, remove the trailing blank lines.
    >+, |+: "keep": keep the line feed, keep trailing blank lines.